doc/engsys_resource_management.md

```diff
@@ -0,0 +1,109 @@
+- The resource group contains a tag with the name `DeleteAfter` and an [ISO8601 formatted date value](https://www.iso.org/iso-8601-date-and-time-format.html)
+  (see below for examples), where the date is greater than the current date and not greater than 7 days in the future.
+    - If the `DeleteAfter` value is in the past or greater than 7 days in the future at the time the cleanup pipeline runs, it will be deleted.
+      The cleanup pipeline runs multiple times daily, starting at 12am pacific time (UTC-8). See the [schedule](https://dev.azure.com/azure-sdk/internal/_apps/hub/ms.vss-ciworkflow.build-ci-hub?_a=edit-build-definition&id=1357&view=Tab_Triggers).
+    - If you have a resource group for which you would like to extend the lifetime, update the `DeleteAfter` tag to a
+      future date to renew the lease.
+    - Resource groups which do not contain a `DeleteAfter` tag will have one added for a 24 hour duration
+      to mark them for deletion and give any test pipelines that are actively using the resources time to
+      complete.
+    - Valid date tag format: `DeleteAfter: 2022-01-29T00:35:48.9372617Z`
+      Example extending resource group lease by three days:
+      ```
+      # Test Resources Script
+      # Update default resource group created via New-TestResources.ps1 and a default DeleteAfter of 48 hours.
+      ./eng/common/TestResources/Update-TestResources.ps1
+      # Update custom resource group name and/or custom DeleteAfter hours value (max 168).
+      ./eng/common/TestResources/Update-TestResources.ps1 -ResourceGroupName <rg name> -DeleteAfterHours 120
+
+      # azure powershell, set manually
+      Set-AzResourceGroup -Name <group name> -Tag @{ DeleteAfter = [DateTime]::UtcNow.AddDays(3).ToString("o") }
+
+      # azure cli, set manually
+      az group update -g <group name> --tags DeleteAfter=$(date -u +"%Y-%m-%dT%H:%M:%SZ" -d "$(date) + 3 day")
+      ```
+
+For long-lived resources, please also add a resource group tag named `Purpose` describing the purpose of the group.
+
+### Role Assignments
+
+Role assignments should be created at the resource group scope or below, where the resource group follows the above
+guidelines. Subscription-level role assignments may be deleted at any time. Reach out to the EngSys team for exemptions.
+
+## Managing CI Resources in the Test Subscription
+
+This section applies to resource groups located in the `Azure SDK Test Resources` subscription. Developers should not
+create resources for individual testing in this subscription, but sometimes resources may need to be created to host
+static assets, secrets or configuration related to pipeline tests.
+
+Currently all EngSys tooling will inspect resource groups and role assignments only.
+
+### Resource Groups
+
+EngSys will delete all resource groups in the testing subscription DAILY, unless they meet at least one of the following criteria:
```

Comment:
Any reason to not keep the same set of rules for tests and playground? Keeping the note about folks should not create resources in the test resources in the general case.

Reply:
Originally the rules were more different with how resources were cleaned up, etc. I think throughout this PR they have converged a bit, so it might be worth treating them the same.

Reply:
Yes lets eliminate as complexity if possible.